src/sentry/receivers/features.py

```diff
@@ -81,6 +83,14 @@ def record_first_event(project, **kwargs):
     )
 
 
+def record_has_sourcemaps(project):
+    cache_key = "has_sourcemaps:project:{}".format(project.id)
+    if cache.get(cache_key) is None:
```

Comment:
Why do we cache here? Checking the bitflag is cheap, and I assume we're not going to be flip-flopping `has_sourcemaps` on and off?